alacritty.yml

```diff
@@ -25,6 +25,8 @@
   #
   # Specified in number of columns/lines, not pixels.
   # If both are `0`, this setting is ignored.
+  # Because of wide characters, the number of columns must not be 1.
+  # When it is 1 it will be automatically set to 2.
```

Comment:
These sentences should be restructured and combined. Currently it's just very difficult to read.